__TESTS__/unit/config/cloudinaryConfig.test.ts

```diff
@@ -0,0 +1,142 @@
+import CloudinaryConfig from "../../../src/config/CloudinaryConfig";
+import AccountConfig from "../../../src/config/AccountConfig";
+import URLConfig from "../../../src/config/URLConfig";
+import CloudConfig from "../../../src/config/CloudConfig";
+
+describe('Tests for CloudinaryConfiguration', () => {
+  it('Creates a CloudinaryConfig with defaults', () => {
+    const conf = new CloudinaryConfig({});
+
+    expect(conf.account).toEqual({});
+    expect(conf.cloud).toEqual({});
+    expect(conf.url).toEqual({});
+  });
```

Comment:
won't you get an error if you don't provide a cloudName?

Reply:
This is the main "CloudinaryConfig" instance, 

It's expected to have a certain shape (accoumt, cloud, url), but we don't expect anything to be mandatory.